### ISSUE TITLE
_available_tags wrong definition

### ISSUE THREAD

**Issue by TeRacksito** [nextcord/nextcord#1034]
### Summary

_available_tags is wrong defined

### Reproduction Steps

```
tags = nextcord.ForumChannel.available_tags
```

### Minimal Reproducible Code

```python
guild = interaction.guild

forum_channel: nextcord.ForumChannel = guild.get_channel(channel_id)
tags = forum_channel.available_tags
print(tags)

await interaction.response.send_message(content= f"{tags[1].name}")
```

### Expected Results

```
[ForumTag id=1234567890123456789 name='tag_1' moderated=True emoji=<PartialEmoji animated=False name='' id=None>, ForumTag id=1234567890123456789 name='tag_2' moderated=False emoji=<PartialEmoji animated=False name='' id=None>, ForumTag id=1234567890123456789 name='tag_3' moderated=False emoji=<PartialEmoji animated=False name='' id=None>]
```

### Actual Results

```
[ForumTag id=1234567890123456789 name='tag_3' moderated=False emoji=<PartialEmoji animated=False name='' id=None>]
```

### Intents

nextcord.Intents.default() + message_content = True

### System Information

- Python v3.11.0-final
- nextcord v2.4.0-final
- aiohttp v3.8.4
- system info: Windows 10 10.0.19045

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [x] I have removed my token from display, if visible.

### Additional Context

- ### **The solution of this problem.** 

The problem seems to be that a **dictionary comprehension** is being used, which will only allow one key-value to be added to the dictionary. In other words, the value of the "id" key is being **overwritten** each time the for loop is iterated.

The solution would be changing the definiton of `_available_tags` in the module `nextcord/channel.py`.
```
class ForumChannel(abc.GuildChannel, Hashable):
> (...)
> def _update(self, guild: Guild, data: ForumChannelPayload) -> None:
> > (...)
> > self._available_tags: Dict[int, ForumTag] = {}
```

This will be a correct dictionary comprehension implemention:
```
self._available_tags: Dict[int, ForumTag] =  {int(tag.get("id")): ForumTag.from_data(tag) for tag in data.get("available_tags", []) if tag.get("id") is not None} | {}
```

**Comment by TeRacksito** [comment]
### I know this should have been fixed on the pull #1021

But it is not fixed.

**Comment by Skelmis** [comment]
Your reproducible code takes the first tag from the list and prints it, when you don't explicitly take that from the list it is returning all tags.

Code used to test:
```py
@bot.event
async def on_ready():
    forum_channel: nextcord.ForumChannel = await bot.fetch_channel(...)
    tags = forum_channel.available_tags
    print(tags)
```

Closing as unclear and unable to reproduce on master
